### PR TITLE
testing/zerotier-one: Update to 1.2.10

### PR DESCRIPTION
**Pull request by kyleparisi**

### CODE REVIEW

File: testing/zerotier-one/APKBUILD [hunk 1]
```diff
@@ -31,6 +31,6 @@ package() {
 			"$pkgdir"/etc/init.d/$pkgname || return 1
```
Comment (andypost) [inline testing/zerotier-one/APKBUILD]: while you toch this, please modernize package by removing this `return 1`

File: testing/zerotier-one/APKBUILD [hunk 2]
```diff
@@ -28,9 +28,9 @@ package() {
 	cd "$builddir"
 	make DESTDIR="$pkgdir" install || return 1
```
Comment (andypost) [inline testing/zerotier-one/APKBUILD]: Please update second commit with removal all `return 1` here and in `build()`
Also commits should be squashed with message `testing/zerotier-one: upgrade to 1.2.10 and modernize`

**Comment by kyleparisi** [comment]
TIL how to squash commits

**Comment by andypost** [comment]
I did further clean-up on commit - npm is not needed to build & added selftest according to https://github.com/zerotier/ZeroTierOne/blob/master/README.md#build-and-platform-notes

**Comment by kyleparisi** [comment]
Just confirming that `subpackages="$pkgname-doc` will be able to be installed without npm?

**Pull request closed without merging**